migrations/2_deploy_organization.js

```diff
@@ -0,0 +1,97 @@
+// Imports:
+const Avatar = artifacts.require("Avatar.sol");
+const Controller = artifacts.require("Controller.sol");
+
+const GlobalConstraintRegistrar = artifacts.require("GlobalConstraintRegistrar.sol");
+const TokenCapGC = artifacts.require("TokenCapGC.sol");
+
+const GenesisScheme = artifacts.require("GenesisScheme.sol");
+const SchemeRegistrar = artifacts.require("SchemeRegistrar.sol");
+const ContributionReward = artifacts.require("ContributionReward.sol");
+const UpgradeScheme = artifacts.require("UpgradeScheme.sol");
+const SimpleICO = artifacts.require("SimpleICO.sol");
+
+const AbsoluteVote = artifacts.require("AbsoluteVote.sol");
+
+// Instances:
+let AbsoluteVoteInst;
+let schemeRegistrarInst;
+let globalConstraintRegistrarInst;
+let upgradeSchemeInst;
+let ControllerInst;
+let AvatarInst;
+
+// DAOstack ORG parameters:
+const orgName = "Genesis";
```

Comment:
I think the initial migration script should deploy only genesisscheme and the universal schemes.
We might have other script which initiate daostack org (demo)using the genesis scheme..

Reply:
@orenyodfat Can you please create a separate PR for this?  Ideally it will describe all the use cases governing the creation of different migration scripts, and what the scripts should do in those cases (I suspect we'll have a different script every time we deploy a new version of arc contracts where some subset of contracts have changed).

For the purpose of the current PR, the current migration script suffices.

Reply:
ok. lets leave it to another pr.